logstash-core/src/main/java/org/logstash/instrument/witness/Witness.java

```diff
@@ -86,4 +93,43 @@ public PipelineWitness pipeline(String name) {
         return pipelinesWitness.pipeline(name);
     }
 
+    @Override
+    public void genJson(JsonGenerator gen, SerializerProvider provider) throws IOException {
+        new Serializer().innerSerialize(this, gen, provider);
```

Comment:
@jakelandis same here, `Serializer` seems like it should just be a static constant since it has no state